README.md

```diff
@@ -37,8 +37,8 @@ This project contains the following top level components:
 * [OpenTelemetry API](api/):
   * [stable apis](api/all/src/main/java/io/opentelemetry/api/) including `Tracer`, `Span`, `SpanContext`, and `Baggage`
   * [semantic conventions](semconv/) Generated code for the OpenTelemetry semantic conventions.
-  * [context api](api/context/src/main/java/io/opentelemetry/context/) The OpenTelmetry Context implementation.
   * [metrics api](api/metrics/src/main/java/io/opentelemetry/api/metrics/) alpha code for the metrics API.
+* [context api](context/src/main/java/io/opentelemetry/context/) The OpenTelemetry Context implementation.
```

Comment:
this is technically part of the "OpenTelemetry API" even though it's not in the `api` directory. I *think* it should be up in the above list, and included in the list of "stable apis" as well.